### PR TITLE
PSP-8337 : UI UX Clean Up - Verbiage - Licence

### PR DESCRIPTION
**Pull request by eddherrera**

### CODE REVIEW

**Comment by devinleighsmith** [comment]
found one extra location:

data.ts 215

also, please update the story to specify that it does not apply to database values at this time.

File: source/frontend/src/features/mapSideBar/router/MapRouter.tsx [hunk 1]
```diff
@@ -294,8 +294,8 @@ export const MapRouter: React.FunctionComponent = memo(() => {
           </LeaseContextProvider>
         )}
         claim={Claims.LEASE_VIEW}
-        key={'LeaseLicense'}
-        title={'Lease / License File'}
+        key={'LeaseLicence'}
```
Comment (devinleighsmith) [inline source/frontend/src/features/mapSideBar/router/MapRouter.tsx:297]: careful changing the keys.

**Review by devinleighsmith — APPROVED** [review]

File: source/frontend/src/features/mapSideBar/property/tabs/propertyDetailsManagement/detail/summary/PropertyManagementDetailView.tsx [hunk 2]
```diff
@@ -48,7 +48,7 @@ export const PropertyManagementDetailView: React.FC<IPropertyManagementDetailVie
           }
         />
       </SectionField>
-      <SectionField label="Lease/Licensed">
+      <SectionField label="Lease/Licenced">
```
Comment (asanchezr) [inline source/frontend/src/features/mapSideBar/property/tabs/propertyDetailsManagement/detail/summary/PropertyManagementDetailView.tsx:51]: sorry to be picky here - **licensed** (as a verb) is the correct work in British/Canadian English. 
Please change to **Licence** (noun) or **Licensed** (verb)

https://ontariotraining.net/word-choice-license-versus-licence/#:~:text=Examples%20(British%20and%20Canadian%20English)&text=(Licence%20is%20a%20noun.),adjectives%2C%20licenced%20may%20be%20used.

File: source/frontend/src/features/leases/add/ReferenceSubForm.tsx [hunk 3]
```diff
@@ -7,11 +7,11 @@ import * as Styled from './styles';
 const ReferenceSubForm: React.FunctionComponent = () => {
   return (
     <Section header="Documentation">
-      <SectionField label="Physical lease/license exists" contentWidth="5">
-        <InlineYesNoSelect field="hasPhysicalLicense" />
+      <SectionField label="Physical lease/licence exists" contentWidth="5">
+        <InlineYesNoSelect field="hasPhysicalLicence" />
```
Comment (asanchezr) [inline source/frontend/src/features/leases/add/ReferenceSubForm.tsx:11]: hmm - this field "hasPhysicalLicence" and the one a couple lines below have been renamed here, but I don't see the corresponding change in the form model class. Could you please check? I wouldn't want this to cause bugs when adding leases.

If you feel the need to modify the frontend model then perhaps the backend models need to be adjusted as well. However I'd leave the fields as-is and only change verbiage

File: source/frontend/src/features/mapSideBar/router/MapRouter.tsx (hunk 1 above)
Comment (eddherrera) [reply source/frontend/src/features/mapSideBar/router/MapRouter.tsx:297]: leaving as it was before.

File: source/frontend/src/features/mapSideBar/property/tabs/propertyDetailsManagement/detail/summary/PropertyManagementDetailView.tsx (hunk 2 above)
Comment (eddherrera) [reply source/frontend/src/features/mapSideBar/property/tabs/propertyDetailsManagement/detail/summary/PropertyManagementDetailView.tsx:51]: Updated

File: source/frontend/src/features/mapSideBar/property/tabs/propertyDetailsManagement/update/summary/PropertyManagementUpdateForm.tsx [hunk 4]
```diff
@@ -57,7 +57,7 @@ export const PropertyManagementUpdateForm = React.forwardRef<
                   options={purposeTypeOptions}
                 />
               </SectionField>
-              <SectionField label="Lease/Licensed">
+              <SectionField label="Lease/Licenced">
```
[the remark this answers is not in the document]
Comment (eddherrera) [reply source/frontend/src/features/mapSideBar/property/tabs/propertyDetailsManagement/update/summary/PropertyManagementUpdateForm.tsx:60]: Updated

File: source/frontend/src/features/leases/add/ReferenceSubForm.tsx (hunk 3 above)
Comment (eddherrera) [reply source/frontend/src/features/leases/add/ReferenceSubForm.tsx:11]: the field prop was not intended to be changed. rolled back.

**Pull request merged**